scanpy/tests/external/test_scrublet.py

```diff
@@ -24,6 +26,52 @@ def test_scrublet():
     assert adata.obs["predicted_doublet"].any(), "Expect some doublets to be identified"
 
 
+def test_scrublet_params():
+    """
+    Test that Scrublet args are passed.
+
+    Check that changes to parameters change scrublet results.
+    """
+    pytest.importorskip("scrublet")
+
+    adata = sc.datasets.pbmc3k()
```

Comment:
As written, this is the longest running test on our CI at 35 seconds.

Reducing the size of the data should make this go much faster.

```suggestion
    # Reduce size of input for faster test
    adata = sc.datasets.pbmc3k()[:500].copy()
    sc.pp.filter_genes(adata, min_counts=100)
```

Reply:
Got it - thanks.